requests/cookies.py

```diff
@@ -242,19 +241,11 @@ def items(self):
 
     def list_domains(self):
         """Utility method to list all the domains in the jar."""
-        domains = []
-        for cookie in iter(self):
-            if cookie.domain not in domains:
-                domains.append(cookie.domain)
-        return domains
+        return list(set([cookie.domain for cookie in iter(self)]))
```

Comment:
How is this reducing readability @kennethreitz ? Can you please elaborate? This is a classic example of "Complex is better than complicated" according to Zen of Python"